Dockerfile

```diff
@@ -29,21 +29,27 @@ RUN apt-get update && apt-get install -y \
 # to download mender-artifact
     wget \
 # to download mender-convert and U-Boot sources
-    git \
-# To compile Mender client, U-boot or GRUB
-    gcc-arm-linux-gnueabihf
+    git
 
 # Disable sanity checks made by mtools. These checks reject copy/paste operations on converted disk images.
 RUN echo "mtools_skip_check=1" >> $HOME/.mtoolsrc
 
+# To provide support for Raspberry Pi Zero W a toolchain tuned for ARMv6 architecture must be used.
+# https://tracker.mender.io/browse/MEN-2399
+# Assumes $(pwd) is /
+RUN wget -nc -q https://toolchains.bootlin.com/downloads/releases/toolchains/armv6-eabihf/tarballs/armv6-eabihf--glibc--stable-2018.11-1.tar.bz2 \
+    && tar -xjf armv6-eabihf--glibc--stable-2018.11-1.tar.bz2 \
+    && rm armv6-eabihf--glibc--stable-2018.11-1.tar.bz2 \
+    && echo export PATH=$PATH:/armv6-eabihf--glibc--stable-2018.11-1/bin >> /root/.bashrc
+
 RUN wget -q -O /usr/bin/mender-artifact https://d1b0l86ne08fsf.cloudfront.net/mender-artifact/$MENDER_ARTIFACT_VERSION/mender-artifact \
     && chmod +x /usr/bin/mender-artifact
 
 # Build liblzma from source
 RUN wget -q https://tukaani.org/xz/xz-5.2.4.tar.gz \
     && tar -C /root -xzf xz-5.2.4.tar.gz \
     && cd /root/xz-5.2.4 \
-    && ./configure --host=arm-linux-gnueabihf --prefix=/root/xz-5.2.4/install \
+    && ./configure --host=arm-buildroot-linux-gnueabihf-gcc --prefix=/root/xz-5.2.4/install \
```

Comment:
The `-gcc` is not necessary here, only the prefix of the toolchain

Honestly, I think this --host option is ignored...

Reply:
Please check now.

Reply:
:+1: